core/lib/spree/app_configuration.rb

```diff
@@ -213,6 +213,12 @@ class AppConfiguration < Preferences::Configuration
     #   @return [Boolean] (default: +true+)
     preference :raise_with_invalid_currency, :boolean, default: true
 
+    # @!attribute [rw] redirect_back_on_unauthorized
+    #   Whether to try to redirect users back when they try to access
+    #   unauthorized routes, before redirect them to /unauthorized.
+    #   @return [Boolean] (default: +false+)
+    preference :redirect_back_on_unauthorized, :boolena, default: false
```

Comment:
This should be `:boolean`

Reply:
Wow, and it's working?

Reply:
🤷‍♂️

Reply:
It works because that information is used for [typecasting](https://github.com/solidusio/solidus/blob/16ed773a6554782e33476e2ffc41b70320ddb46e/core/lib/spree/preferences/preferable.rb#L144). In that case, no typecasting would occur as the type would not be recognized.